### DISCUSSION TITLE
DV-Multiplayer v0.1.7 Beta

### DISCUSSION THREAD

**Discussion by AMacro** [AMacro/dv-multiplayer#18] [Announcements]
Change  Log

UI
- Fixed bug preventing previous saves from being loaded in the session manager
- Fixed bug that stopped 'selected' item from highlighting in server browser
- Added 'connecting' dialogue and feedback

Network
- Added a connection sequence, server browser will now attempt to connect on IPv6 if available, then fall back to IPv4 if available.
- Updated lobby server to work correctly with IPv6 and IPv4
- Groundwork for NAT/firewall punching has been implemented, but not yet complete

Game
- Fixed GetNetID() exception spamming 
- Implemented fix for race condition when a car is deleted/expired just as the player boards
- Implemented fix for car spawning desync issues when a player enters/exits a car (player coordinates to be temporarily corrupted)
- Implemented a /kick command (not currently listed in /help)
- Fixed an issue where windows on locomotives would break when they were supposed to be repaired

<hr /><em>This discussion was created from the release <a href='https://github.com/AMacro/dv-multiplayer/releases/tag/v0.1.7-Beta'>DV-Multiplayer v0.1.7 Beta</a>.</em>

**Comment by Strider0ne** [comment]
Suppose I could have written here, either way can't join the server my friend makes. Public, no password.

**Reply by AMacro** [reply]
Hey @Strider0ne,
Couple of questions:

~~1. Could you see his server in the server browser?~~ Edit: saw your other thread
2. Are you able to confirm he had opened the appropriate port on his router (default is UDP 7777)?
3. Have you checked both of you had allowed DerailValley in your firewall?

If you're on Discord, jump into [here](https://discord.com/channels/332511223536943105/1234574186161377363) and we can test if you can connect to my server to rule out your end

**Reply by Strider0ne** [reply]
I mentioned it, he said his router doesn't allow it. 
Derail valley is allowed through my firewall now, don't know if it's the same for him. 
The discord link doesn't seem to work.

**Reply by Strider0ne** [reply]
I got it to work, had to copy the link to my app. Didn't want to work through the browser.
